storm-client/src/jvm/org/apache/storm/tuple/Values.java

```diff
@@ -23,9 +23,13 @@ public Values() {
     }
 
     public Values(Object... vals) {
-        super(vals.length);
-        for (Object o : vals) {
-            add(o);
+        super(vals != null ? vals.length : 0);
```

Comment:
I think this can be `super(vals != null ? vals.length : 1);` since `if (vals==null)` we are going to add `null` to it. The length will be 1.

Reply:
Making change